packages/backend/types/auth/triggers/index.ts

```diff
@@ -0,0 +1,8 @@
+export {
```

Comment:
do we need a submodule export in the package.json of this file to be able to access these types?

Also, we should import them all into an `index.internal.ts` file and point the API extractor to that file for this package so that the API file is capturing all of the surface area.

Reply:
Done!